src/lando/api/views.py

```diff
@@ -148,3 +155,22 @@ class hg2gitCommitMapView(CommitMapBaseView):
     """Return corresponding CommitMap given an hg hash."""
 
     scm = SCM_TYPE_HG
+
+
+class PullRequestAPIView(APIView):
+    """Handle pull requests in the API."""
+
+    def get(self, request: WSGIRequest, repo_name: str, number: int) -> JsonResponse:
+        """Return a serialized JSON representation of a pull request."""
+        target_repo = Repo.objects.get(name=repo_name)
+        client = GitHubAPIClient(target_repo)
+        pull_request = PullRequest(client.get_pull_request(number))
+        return JsonResponse(pull_request.serialize(), status=200)
+
+
+class LandingJobAPIView(View):
+    """Handle landing jobs in the API."""
+
+    def post(self, request: WSGIRequest, *args, **kwargs):
+        """Placeholder for creating new landing jobs."""
+        pass
```

Comment:
Oh! I discovered recently that if something empty has a docstring, it doesn't need a pass. The docstring is enough to satisfy the parser that there is a body.

The `pass` doesn't hurt, of course, and I still forget about it, but heh (: